php/apache2/install.sls

```diff
@@ -1,18 +1,23 @@
-{% from "php/map.jinja" import php with context %}
+{%- set tplroot = tpldir.split('/')[0] %}
+{%- from tplroot ~ "/map.jinja" import php with context %}
+{%- from tplroot ~ "/libtofs.jinja" import files_switch with context %}
 
-{% set state = 'apache2' %}
-{% include "php/installed.jinja" %}
+{%- set state = 'apache2' %}
+{%- include tplroot ~ "/installed.jinja" %}
 
-{% if grains['os_family'] == "FreeBSD" %}
+{%- if grains['os_family'] == "FreeBSD" %}
 {{ php.lookup.apache2.module_config }}:
   file.managed:
-    - source: salt://php/apache2/files/mod_php.conf.jinja
+    - source: {{ files_switch(['mod_php.conf.jinja'],
+                               'php_apache2_module_config',
+                               use_subpath = '/apache2'
```

Comment:
```suggestion
                              use_subpath=True
```

* This is the way to use the `use_subpath` parameter (more info in `libtofs.jinja` or the TOFS document).

Reply:
Done